core/src/main/java/com/graphhopper/reader/osm/OSMReader.java

```diff
@@ -98,6 +109,15 @@ public OSMReader(BaseGraph baseGraph, TagParserManager tagParserManager, OSMRead
         simplifyAlgo.setElevationMaxDistance(config.getElevationMaxWayPointDistance());
         turnCostStorage = baseGraph.getTurnCostStorage();
 
+        try {
+            Path filePath = Paths.get("logs/ways_dump.ldgeojson");
```

Comment:
Can you make this configurable to match the current_log_filename in the config? Also preferably off by default so this big file isn't sitting there if you're not going to use it